frontend/src/models/licensing.ts

```diff
@@ -169,12 +173,14 @@ export function selectLicenses(state: ApplicationState) {
 }
 
 export function selectLicenseIndicator(state: ApplicationState) {
+  const { informed } = state.licensing
   let indicators = 0
   const { licenses } = selectLicenses(state)
   for (var license of licenses) {
     const { noticeType } = selectLicense(state, license?.plan.product.id)
     if (noticeType) indicators++
   }
+  if (informed) return 0
```

Comment:
Might be better to just return before doing all the selection logic for best performance.